# Analytical Code for Spatial Transcriptomics Analysis

This repository contains analytical code used to perform spatial transcriptomics analyses for the **Dual Spatial Transcriptomics Study**. Scripts and Jupyter notebooks were developed to process, visualize, and interpret spatially resolved hostâ€“pathogen transcriptomic data.

## Data Availability
Raw sequencing data are available in the NCBI Gene Expression Omnibus (GEO) under accession **PRJNA1202130**:  
https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1202130

## Computational Workflow
A complete, executable workflow with Jupyter notebooks is provided at:  
https://github.com/modernatx/spatial_RNAseq_analyses

### Included Notebooks
- **Bacterial_target_expression_plot.ipynb**: visualization of pathogen transcript abundance across spatial regions of interest (ROIs).
- **GO_enrichment_host_response.ipynb**: Gene Ontology enrichment analysis of host transcriptomic responses.
- **Predictive_models.ipynb**: ridge regression models predicting bacterial load from host gene expression profiles.

## Contact
Hao Zhou  
hao.zhou@ku.ac.ae
